src/style.css

```diff
@@ -1,3 +1,109 @@
 body {
   margin: 0;
+  font-family: "Roboto", "Helvetica", sans-serif;
+}
+
+.products {
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  margin: auto;
+}
+
+.products__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  width: 166px;
+  margin: 34px auto 0;
+  color: rgba(6, 11, 53, 1);
+}
+
+.products__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: rgba(97, 96, 112, 1);
+  margin: 5px 17px 0;
+}
+
+.products__img {
+  width: 160px;
+  height: 134px;
+  margin: 33px 0 0 20px;
+}
+
+.products__stars {
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 17px 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.stars__star {
+  background: url(../src/images/star.svg) no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.products__number {
+  font-size: 10px;
+  line-height: 14px;
+  text-align: center;
+  margin-left: 18px;
+}
+
+.stars--4 > .stars__star:nth-child(-n + 4) {
+  background: url(../src/images/star-active.svg) no-repeat;
+  background-position: center;
+}
+
+.products__size {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 17px;
+}
+
+.products__price {
+  color: rgba(97, 96, 112, 1);
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.products__text {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: rgba(6, 11, 53, 1);
+}
+
+.products__link {
+  width: 166px;
+  height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Just use padding